examples/offline_inference/disaggregated_prefill.py

```diff
@@ -0,0 +1,98 @@
+"""
+This file demonstrates the example usage of disaggregated prefilling
+We will launch 2 vllm instances (GPU 0 for prefill and GPU 1 for decode),
+and then transfer the KV cache between them.
+
+Learn more about Ray Data in https://docs.ray.io/en/latest/data/data.html
+"""
+import os
+import time
+from multiprocessing import Event, Process
+
+from vllm import LLM, SamplingParams
+from vllm.config import KVTransferConfig
+
+
+def run_prefill(prefill_done):
+    # We use GPU 0 for prefill node.
+    os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+
+    # The prefill node receives two requests, while the decode node receives
+    # three requests. So the decode node will only receive the KV Cache for
+    # requests 1 and 3. The decode node will use the KV Cache of requests 1
+    # and 3 and do prefilling on request 2.
+    prompts = [
+        "Hello, my name is",
+        # "Hi, your name is", # To trigger partial prefill of batched requests
```

Comment:
"partial prefill" is not the right term for this case.

```suggestion
        # "Hi, your name is", # The decode node will actually "prefill" this request.
```

Reply:
Thanks for your fix. I've changed this comment.